.github/workflows/daily_reports.yaml

```diff
@@ -157,7 +158,6 @@ jobs:
       - name: Post results to levm slack channel
         env:
           url: ${{ secrets.LEVM_SLACK_WEBHOOK }}
-        run: sh .github/scripts/publish_loc.sh
```

Comment:
why are we removing this line? isn't it needed?

Reply:
Yes! That's why I was getting an error. I deleted the line by mistake.

Reply:
Fixed here: https://github.com/lambdaclass/ethrex/pull/1433/commits/ef7bd7112ba770b919bc9e9eefbb3aac0f1c5db4